src/main/java/core/basesyntax/strategy/impl/ReturnOperationHandler.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+
+public class ReturnOperationHandler implements OperationHandler {
+    private final FruitStorage fruitStorage;
+
+    public ReturnOperationHandler(FruitStorage fruitStorage) {
+        this.fruitStorage = fruitStorage;
+    }
+
+    @Override
+    public void apply(FruitTransaction transaction) {
+        fruitStorage.addFruit(transaction.getFruit(), transaction.getQuantity());
```

Comment:
Ensure that the `addFruit` method in `FruitStorage` accumulates the quantity if the fruit already exists, rather than overwriting it. This will ensure that returned fruits are correctly added to any existing stock.